esp/platform/application_config.cpp

```diff
@@ -214,6 +219,27 @@ void bindService(IPropertyTree *legacyEsp, IPropertyTree *app, const char *servi
         if (authenticate)
             bindAuthResources(authenticate, app, service, auth);
     }
+
+    // Add contents of applicationBindingInfo to static EspBinding element by service name
+    xpath.setf("applicationBindingInfo/%s", service);
+    IPTree *branch = app->queryBranch(xpath.str());
+    if (branch)
+    {
+        Owned<IAttributeIterator> aiter = branch->getAttributes();
+        ForEach (*aiter)
+        {
+            const char *name = aiter->queryName();
+            bindingEntry->addProp(name+1, aiter->queryValue());
```

Comment:
Note that this means we can't easily define attributes to be added, but because of the way Yaml ptree works if you make attributes easy to add, then elements are more verbose:

So, either:
```
MyYaml:
  notanattribute: 123
  element: abc
```
or
```
MyYaml:
  attribute: 123
  element: [ abc ]
```
Having to do all scalar elements like that would be cumbersome.  And the need for custom top level binding attributes may be rare.

Reply:
Actually, the real problem is that this will be inconsistent with the ptree added below.  That is, at this top level simple scalars will be elements, but when you do the ```bindingEntry->addPropTree(ele.queryName(), LINK(&ele));``` below they will be attributes.

Reply:
Will take some playing with to make sure it's usable.  But I would think, just replace this whole section that is merging the attributes and elements with a call to mergePTree(target, toMerge);

If users want simple scalar elements they will have to do.
```
MyYaml:
  attribute: 123
  element: [ abc ]
 ```